Classes/Domain/Model/Product.php

```diff
@@ -979,6 +979,20 @@ public function updateAttributeValue(Attribute $attribute, $value): void
         $this->attributeValueUpdater->update($this, $attribute, $value);
     }
 
+    /**
+     * Return listing images if found.
+     *
+     * @return array
+     */
+    public function getListImages(): array
+    {
+        // @codingStandardsIgnoreStart
+        return empty($this->findImagesByType(Image::LISTING_IMAGE))
+            ? [$this->getMainImage()]
+            : $this->findImagesByType(Image::LISTING_IMAGE);
```

Comment:
I wonder if this will solve the CS error:

```suggestion
        return empty(
             $this->findImagesByType(Image::LISTING_IMAGE))
            ? [$this->getMainImage()]
            : $this->findImagesByType(Image::LISTING_IMAGE
        );
```